app/trimmer_decorator.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './decorator'
+class TrimmerDecorator < BaseDecorator
+  attr_accessor :person
+
+  def initialize(person)
+    super()
```

Comment:
- [ ] [OPTIONAL] Kindly remove `super( )` in the `initialize` method inside `TrimmerDecorator` as it will cause an error when you try to create a new object of `TrimmerDecorator`.